docs/plugins/analysis-phonetic.asciidoc

```diff
@@ -17,9 +17,9 @@ The `phonetic` token filter takes the following settings:
 `encoder`::
 
     Which phonetic encoder to use.  Accepts `metaphone` (default),
-    `doublemetaphone`, `soundex`, `refinedsoundex`, `caverphone1`,
+    `double_metaphone`, `soundex`, `refined_soundex`, `caverphone1`,
```

Comment:
Maybe also change "refinedsoundec" in plugins/analysis-phonetic/src/test/resources/org/elasticsearch/index/analysis/phonetic-1.yml then